content/docs/instrumenting/exporters.md

```diff
@@ -121,3 +122,4 @@ practices.
    * Go: [go-metrics instrumentation library](https://github.com/armon/go-metrics)
    * Go: [gokit](https://github.com/peterbourgon/gokit)
    * Java/JVM: [Hystrix metrics publisher](https://github.com/soundcloud/prometheus-hystrix)
+   * Rust: [rust-prometheus](https://github.com/pingcap/rust-prometheus/)
```

Comment:
That's a client library and already documented on https://prometheus.io/docs/instrumenting/clientlibs/